database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java

```diff
@@ -171,17 +222,24 @@ public void prepareRun(BatchSinkContext context) {
     }
 
     // Get Hadoop configuration object
-    Configuration configuration = configAccessor.getConfiguration();
+//    Configuration configuration = configAccessor.getConfiguration();
```

Comment:
why this is commented?

Reply:
The configuration are being set in another way and it was not required anymore. Therefore, commented out the code.

Reply:
remove it from comments

Reply:
resolved